cmd/bnscli/clitests/README.md

```diff
@@ -4,6 +4,45 @@ This is a set of tests executing shell scripts and ensuring the output is as
 expected. Tests expect `bnscli` binary to be present in one of the `$PATH`
 directories.
 
+## Examples
+
+This directory largely serves as self-documenting examples of how to create
+transactions. These can be seen below:
+
+* [Create Multisig](./attach_multisig_id.test)
+* [Create batch of send tx](./batch.test)
+* [Governance proposal with batch command](./batch_proposal.test)
+
+**TODO** Is this useful?
```

Comment:
I think TODO should be either removed or formulated in a way that is actionable and without using the word _TODO_ :wink: